files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -76,6 +76,9 @@ def update_filter_metadata(metadata_to_update):
                 filter_metadata.append({"key": filter_field_key, "value": filter_field_value})
     filter_metadata = pydash.uniq(filter_metadata)
     metadata_to_update["advSearchFilters"] = filter_metadata
+    metadata_to_update["tags"] = [
+        {"name": f["value"], "category": f["key"]} for f in filter_metadata
```

Comment:
this section should preservs existing tags in `Data Repository` category?

Reply:
No, for this section I assumed that there weren't any existing tags to save. I can implement the logic from the migration script to save any existing `Data Repository` tags.